apps/api_web/test/api_web/controllers/schedule_controller_test.exs

```diff
@@ -300,6 +300,31 @@ defmodule ApiWeb.SchedulerControllerTest do
       assert index_data(conn, put_in(params["max_time"], "5:30")) == []
     end
 
+    test "doesn't excludes arrival and departure times for older API versions", %{conn: conn} do
```

Comment:
This test doesn't make sense anymore.

Reply:
Removed. I've added similar tests to `schedule_view_test.exs` yesterday.